frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/pvc-delete-extension.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { k8sKill, K8sResourceKind } from '@console/internal/module/k8s';
+import { DataVolumeModel } from '../../models';
+
+export const killCDIBoundPVC = (pvc: K8sResourceKind) => k8sKill(DataVolumeModel, pvc);
+
+export const PVCDeleteAlertExtension: React.FC<PVCDeleteAlertExtension> = ({ pvc }) => (
```

Comment:
you should also consider that the extension would define `alertTitle: string`, `alertBody: LazyLoader<{pvc: PersistentVolumeClaimKind}>`, `alertType: AlertVariant`. 

It makes the extension less error prone, you dont have to specify className, isInline etc..all would be handled by pvc delete modal